misc/multistream-select/src/listener_select.rs

```diff
@@ -72,7 +73,14 @@ where
     // TODO: It would be nice if eventually N = Protocol, which has a
     // few more implications on the API.
     protocols: SmallVec<[(N, Protocol); 8]>,
-    state: State<R, N>
+    state: State<R, N>,
+    /// Whether the last message sent was a protocol rejection (i.e. `n/a`).
```

Comment:
I am not sure what is meant with `n/a`, did you mean to write `na\n`?

```suggestion
    /// Whether the last message sent was a protocol rejection (i.e. `na\n`).
```